Dockerfile

```diff
@@ -20,26 +20,25 @@ RUN DEBIAN_FRONTEND='noninteractive' \
 
 RUN apt-get clean
 
-# Install latest python
-ADD ./docker/install_python.sh /install_python.sh
-RUN chmod 755 /install_python.sh
-RUN /install_python.sh
-
-# Add requirements to docker
-ADD ./requirements/base.txt /requirements.txt
-RUN pip install -r /requirements.txt
-
-# Add project directory to docker
+# Add project to container
 ADD . /home/app
-RUN rm -rf /home/app/.git
-RUN chmod 755 /home/app/docker/collectstatic.sh
-RUN /home/app/docker/collectstatic.sh
-RUN  chown -R app: /home/app
 
 # Set correct environment variables.
 ENV HOME /home/app
-WORKDIR /home/app
 ENV APP_HOME /home/app
+WORKDIR /home/app
+
+RUN rm -rf /home/app/.git
+
+# Install latest python
+RUN docker/install_python.sh
+
+RUN pip install -r requirements.txt
```

Comment:
Similar to the python install, could this happen before the "copy the code" part, please?